source/2019-12-28-countdown-to-the-new-year-ember-changeset.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Countdown to The New Year - Ember Changeset
+author: Scott Newcomer
+tags: 2019, Addons, DecEmber
+alias: 2019-12-28-countdown-to-the-new-year-ember-changeset.md
+responsive: true
+---
+
+**This is the 28th in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 28
+
+For the 28th edition of **Countdown to The New Year** we're going to take a
+look at [`ember-changeset`](https://emberobserver.com/addons/ember-changeset).
+
+### Ember Changeset - What It Does
+Managing data associated with forms can sometimmes be unwieldy. As a result, we have borrowed concepts from the Elixir [community](https://hexdocs.pm/ecto/Ecto.Changeset.html#content) to give you the ability to prevent mutating the underlying object until the last possible moment when you have decided to apply the changes and the data is valid. This functional approach to validations has proved to be quite useful.
+
+This addon works in conjunction with [`ember-changeset-validations`](https://github.com/poteto/ember-changeset-validations/) to provide you with out of the box validations to validate the format of a certain input, ensure passwords pass a minimum security threshold or even let you build your own custom validator!
+
+### Ember Changeset - Why I Like It
```

Comment:
What do you think about the following? This should resolve the markdown lint error on CI:

```suggestion
### Why I Like It

```